inframanager/config.yaml

```diff
@@ -1,8 +1,22 @@
-GrpcServer:
+P4RuntimeServer:
   addr: localhost:9559
+# Possible authentication types
+#  Insecure
+#  MutualTLS
+  auth: Insecure
 
 GnmiServer:
   addr: localhost:9339
+# Possible authentication types
+#  Insecure
+#  MutualTLS
+  auth: Insecure
+
+# Possible authentication types
+#  Insecure
+#  ServersideTLS
+#  MutualTLS
+InfraManagerAuth: MutualTLS
 
```

Comment:
Change it to
Inframanager:
  ConnectionType: Secure/Insecure
  EncryptionMode: mTLS